src/collection-view.js

```diff
@@ -339,7 +339,7 @@ const CollectionView = Backbone.View.extend({
 
   _sortModelsBy(models, comparator) {
     if (typeof comparator === 'string') {
-      return _.sortBy(models, function(model) {
+      return _.sortBy(models, (model) => {
```

Comment:
@yethee of course, but i tried to write in one style like it was done before in other places
e.g.
```js
models = _.filter(models, (model, index) => {
        return this._shouldAddChild(model, index);
      });
```